components/Cards/SchoolCard/__tests__/BackSchoolCard.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import BackSchoolCard from '../BackSchoolCard';
+
+const locations = [
+  {
+    va_accepted: true,
+    address1: '825 Battery Street',
+    address2: '3rd Floor',
+    city: 'San Francisco',
```

Comment:
Let's provide `state` if city is provided. This one requires a bit of context, but basically the back-end will either not provide us EITHER of city or state, or they provide both. Never one separately. It's tough to communicate that with prop-types, but make sure the test data reflects such.